core/state/state_object.go

```diff
@@ -194,6 +204,28 @@ func (s *StateObject) GetState(db Database, key common.Hash) common.Hash {
 	return s.GetCommittedState(db, key)
 }
 
+func (s *StateObject) getStorageKey(key common.Hash) (common.Hash, bool) {
```

Comment:
This is quite confusing for me, a function name `getStorageKey`, while it is parameter is a `key`, the return is a `value`?